models/balancer/arbitrum/balancer_v2_arbitrum_transfers_bpt.sql

```diff
@@ -1,6 +1,7 @@
 {{
     config(
-        schema='balancer_v2_arbitrum',
+        schema = 'balancer_v2_arbitrum',
+        tags = ['dunesql'],
         alias = alias('transfers_bpt'),
         partition_by = ['block_date'],
```

Comment:
we're looking to increase our partition size -- can you add a `block_month` column to the model, just like `block_date`, then update value here?

note, this would apply anywhere you partition. you could also explore removing partition altogether if the dataset isn't v large